Find a functionally equivalent code:android.app.Activity.onPanelClosed:COMMENT
Method Modifier: public      
Comment:/**
 * Default implementation of
 * {@link android.view.Window.Callback#onPanelClosed(int, Menu)} for
 * activities. This calls through to {@link #onOptionsMenuClosed(Menu)}
 * method for the {@link android.view.Window#FEATURE_OPTIONS_PANEL} panel,
 * so that subclasses of Activity don't need to deal with feature codes.
 * For context menus ({@link Window#FEATURE_CONTEXT_MENU}), the
 * {@link #onContextMenuClosed(Menu)} will be called.
 */

@@ -8,14 +8,14 @@
             onContextMenuClosed(menu);
             break;
         case Window.FEATURE_ACTION_BAR:
-            initActionBar();
+            initWindowDecorActionBar();
             mActionBar.dispatchMenuVisibilityChanged(false);
             break;
     }

